cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,73 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() =>{
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
-});
+
+  it('Assert the pagination and rows count selection', () => {
+    cy.get('.pagination-bottom').should('exist');
+    cy.get('.-next').click();
+    cy.get('.-previous').click();
+    cy.get('[aria-label="rows per page"]').should('exist');
+    cy.get('[aria-label="rows per page"]').select('5 rows');
+  });
+
+  it('Should add a new employee and delete him', () => {
+    cy.get('#addNewRecordButton.btn.btn-primary').click();
+    cy.findByPlaceholder('First Name').type(user.firstName);
+    cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.findByPlaceholder('name@example.com').type(user.email);
+    cy.findByPlaceholder('Age').type(user.age);
+    cy.findByPlaceholder('Salary').type(user.salary);
+    cy.findByPlaceholder('Department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#delete-record-4').click();
```

Comment:
```suggestion
    cy.get('#delete-record-4').click();
```
you do not assert that user is really deleted. You just click the delete button